chia/data_layer/data_layer.py

```diff
@@ -533,7 +545,7 @@ async def periodically_manage_data(self) -> None:
                     try:
                         await self.update_subscriptions_from_wallet(subscription.tree_id)
                         await self.fetch_and_validate(subscription.tree_id)
-                        await self.upload_files(subscription.tree_id)
+# await self.upload_files(subscription.tree_id)
```

Comment:
This now is for test. If works, is necessary to change this to not create files prior to the snapshot